lib/view/authentication/login_screen.dart

```diff
@@ -0,0 +1,253 @@
+                      padding: EdgeInsets.symmetric(
+                        horizontal: 25,
+                        vertical: 20,
+                      ),
+                      decoration: BoxDecoration(
+                        borderRadius: BorderRadius.circular(8),
+                        color: Color(0xffffffff).withOpacity(0.05),
+                      ),
+                      child: RichText(
+                        text: TextSpan(
+                          children: <TextSpan>[
+                            TextSpan(
+                              text: "Don't Have an account?, ",
+                              style: TextStyle(
+                                fontWeight: FontWeight.w500,
+                              ),
+                            ),
+                            TextSpan(
+                              text: ' Sign Up',
+                              style: TextStyle(
+                                color: Color(0xff8DEBFF),
+                              ),
+                            ),
+                          ],
+                        ),
+                      ),
+                    ),
+                  ),
+                  SizedBox(
+                    height: 20,
+                  ),
+                ],
+              ),
+              Footer(),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class AuthIcons extends StatelessWidget {
+  const AuthIcons({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        GestureDetector(
+          onTap: () {
+            // Google Sign In method
+          },
+          child: SvgPicture.asset(
+            'assets/svg/google.svg',
+            width: 40,
+          ),
+        ),
+        SizedBox(
+          width: 30,
+        ),
+        GestureDetector(
+          onTap: () {
+            // Github auth
+          },
+          child: Image.asset(
+            "assets/images/github.png",
+            width: 40,
+          ),
+        ),
+      ],
+    );
+  }
+}
+
+class OrLine extends StatelessWidget {
+  const OrLine({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 25.0),
+      child: Row(
+        children: [
+          Expanded(
+              child: Divider(
+            color: Colors.white,
+            thickness: 2,
+          )),
+          Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 12.0),
+            child: Text(
+              "OR",
+              style: MyTextStyles.customStyle(
+                fontsize: 12,
+                fontWeight: FontWeight.bold,
+              ),
+            ),
+          ),
+          Expanded(child: Divider(color: Colors.white, thickness: 2)),
+        ],
+      ),
+    );
+  }
+}
+
+class HeadLine extends StatelessWidget {
```

Comment:
Export entire logo from figma , Don't need to create our own logo headline

Reply:
Done